test/resources/case9_templates/case9-test-policy.yaml

```
@@ -0,0 +1,64 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: case9-test-policy
+spec:
+  remediationAction: inform
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: case9-test-configpolicy
+        spec:
+          remediationAction: inform
+          namespaceSelector:
+            exclude: ["kube-*"]
+            include: ["default"]
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                kind: ConfigMap
+                apiVersion: v1
+                metadata:
+                  name: case9-test-configmap
+                  namespace: test
+                data:
+                  # Configuration values can be set as key-value properties
+                  Clustername: '{{hub .ManagedClusterName hub}}'
+                  Vlanid: |
```

You wouldn't want a new line at the end of the `Vlaidid` value. It's a more realistic test-case.
```suggestion
                  Vlanid: >-
```

---

Actually, I think the `toInt` strips the new line which is why this works.

---

@mprahl : I did try to make the change to  >-  , as its just a tiny thing , but e2e test failed due to it, I know you mentioned about >- not working previously, so I am not sure if its related or something with the combination of >- and toInt. 

Either way, not fair to hold this PR due that ,  IMO,  propagation controller is primarily checking template resolution and propagation here so its  probably ok to have |  ,  Various checking combinations of >- or | or the functions probably more suited to be tested as part of go-template-util test.